src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml

```diff
@@ -4,51 +4,92 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:platformUi="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.15.0"
     Title="SonarLint Connected Mode Migration"
+    HorizontalAlignment="Center"
     ResizeMode="NoResize"
-    SizeToContent="WidthAndHeight">
+    SizeToContent="WidthAndHeight"
+    MinHeight="300"
+    MaxHeight="500"
+    MinWidth="200"
+    MaxWidth="500">
 
     <Window.Resources>
         <Style x:Key="Button_Style" TargetType="Button">
-            <Setter Property="Height" Value="25" />
-            <Setter Property="Width" Value="75" />
+            <Setter Property="MinHeight" Value="25" />
+            <Setter Property="MinWidth" Value="75" />
         </Style>
+
     </Window.Resources>
 
-    <StackPanel Height="235" Width="350">
-        <Separator/>
-        <StackPanel x:Name="StartWindow" Visibility="Visible" Orientation="Vertical" Margin="10,10,10,10" >
-            <TextBlock Height="180" Width="Auto" HorizontalAlignment="Left" VerticalAlignment="Top" >
-                <Run>The storage of Connected Mode has settings has changed, </Run><LineBreak />
-                <Run>and the settings for this project need to be migrate to  the </Run><LineBreak />
-                <Run>new format. See </Run> <Hyperlink> here</Hyperlink> <Run> for more information.</Run><LineBreak />
-                
-                <LineBreak />
+    <Grid TextBlock.FontSize="15">
+        <DockPanel x:Name="StartWindow" Margin="10" LastChildFill="True" Visibility="Visible" >
+            <DockPanel DockPanel.Dock="Bottom">
+                <Button DockPanel.Dock="Left" Content="Cancel" IsCancel="True" HorizontalAlignment="Left"  Style="{DynamicResource Button_Style}" />
+                <Button DockPanel.Dock="Right" Content="Continue" Click="NavigateToMigrationProgressPage" HorizontalAlignment="Right" Style="{DynamicResource Button_Style}"/>
+            </DockPanel>
 
-                <Bold><Run>Before starting:</Run></Bold><LineBreak />
-                <Run>* check that you have no unsaved files</Run><LineBreak />
-                <Run>* if the solution is under source control, check that the </Run><LineBreak/>
-                <Run>  you have no uncommitted changes.</Run><LineBreak />
+            <!-- Last child - fills the body of the panel -->
+            <StackPanel Orientation="Vertical" HorizontalAlignment="Left">
 
-                <LineBreak />
+                <!-- Intro -->
+                <TextBlock TextWrapping="WrapWithOverflow" >
```

Comment:
I split the text into separate logical sections i.e. separate `TextBlock`s. That gives us more control over the spacing/layout of each block.
However, we might not need it. We should revisit it when we look at theming and how the dialog looks at different themed sizes.